vendor/engines/badges_engine/spec/dummy/config/application.rb

```diff
@@ -0,0 +1,27 @@
+require File.expand_path('../boot', __FILE__)
+
+# Pick the frameworks you want:
+require "active_record/railtie"
+require "action_controller/railtie"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.